src/components/combobox/utils.ts

```diff
@@ -30,7 +30,7 @@ export function getDepth(element: HTMLElement): number {
   }
 
   const result = document.evaluate(
-    "ancestor::calcite-combobox-item | ancestor::calcite-combobox-item-group",
+    "ancestor::calcite-combobox-item",
```

Comment:
Does the depth affect the item selector indicator logic? If so, can you test the use case from https://github.com/Esri/calcite-components/issues/2337 to confirm it won't introduce any regressions?

Reply:
Nope, the depth doesn't affect the selector indicator logic. With enhanced UI design , `calcite-combobox-item-group` elements are no longer considered as nested ones similar to dropdown-item-group. The depth will be zero for such items in group. 

Regarding https://github.com/Esri/calcite-components/issues/2337 , it is working as expected.
![ED84E34B-BF16-4B22-8109-182BA149CA73_4_5005_c](https://user-images.githubusercontent.com/88453586/213836688-467c1dbe-246a-4fea-b61b-1bf915c44dbe.jpeg)

Reply:
Awesome. Thanks for the breakdown and for testing.